### PR TITLE
Copy edit 10516 fedora install

### PR DESCRIPTION
**Pull request by SvenDowideit**
This should replace #10516

@fredlf @jamtur01  @lsm5 @vbatts

### CODE REVIEW

File: docs/sources/installation/centos.md [hunk 1]
```diff
@@ -4,48 +4,33 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
...
-To run Docker, you will need [CentOS6](http://www.centos.org) or higher,
-with a kernel version 2.6.32-431 or higher as this has specific kernel
-fixes to allow Docker to run.
+## Kernel support
+
+Currently the CentOS project will only support Docker when running on kernels
+shipped by the distribution. There are kernel changes which will cause issues
+if one decides to step outside that box and run non-distribution kernel packages.
+
+To run Docker on [CentOS-6.5](http://www.centos.org) you will need kernel version
```
Comment (vbatts) [inline docs/sources/installation/centos.md]: if you pin a version, like "6.5" here, you'll be needing update regularly. Something like "centos6 (see site for latest version)" or similar, should allow the docs to stay a bit more timeless

File: docs/sources/installation/centos.md [hunk 2]
```diff
@@ -4,48 +4,33 @@ page_keywords: Docker, Docker documentation, requirements, linux, centos, epel,
...
-this changes the installation instructions slightly between versions. If you
-need the latest version, you can always use the latest binary which works on
-kernel 3.8 and above.
+Docker is supported on the following versions of CentOS:
 
-These instructions work for CentOS 6 and later. They will likely work for
-other binary compatible EL6 distributions such as Scientific Linux, but
-they haven't been tested.
+- [*CentOS 7 (64-bit)*](#installing-docker---centos-7)
+- [*CentOS 6.5 (64-bit)*](#installing-docker---centos-6.5)
```
Comment (vbatts) [inline docs/sources/installation/centos.md]: same version note.

File: docs/sources/installation/centos.md (hunk 1 above)
Comment (vbatts) [inline docs/sources/installation/centos.md]: For instance, the current release is already 6.6

File: docs/sources/installation/rhel.md [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+page_keywords: Docker, Docker documentation, requirements, linux, rhel
 
-# Red Hat Enterprise Linux 7
+# Red Hat Enterprise Linux
 
-**Red Hat Enterprise Linux 7** has [shipped with
+Docker is supported on the following versions of RHEL:
+
+- [*Red Hat Enterprise Linux 7 (64-bit)*](#red-hat-enterprise-linux-7-installation)
+- [*Red Hat Enterprise Linux 6.5 (64-bit)*](#red-hat-enterprise-linux-6.5-installation)
```
Comment (vbatts) [inline docs/sources/installation/rhel.md]: same version nit applies.

File: docs/sources/installation/rhel.md [hunk 4]
```diff
@@ -25,51 +39,44 @@ Enterprise Linux 7](https://access.redhat.com/site/articles/881893)
...
 
 Firstly, you need to install the EPEL repository. Please follow the
 [EPEL installation
 instructions](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
 
-The `docker-io` package provides Docker on EPEL.
+For CentOS-6, there is a package name conflict with a system tray application
+and its executable, so the Docker RPM package was called `docker-io`.
+
+To proceed with `docker-io` installation on CentOS-6, you may need to remove the
```
Comment (vbatts) [inline docs/sources/installation/rhel.md]: Says Centos here. Not sure if that was intentional.

**Comment by vbatts** [comment]
a couple of nits, after that LGTM

File: docs/sources/installation/centos.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/sources/installation/centos.md]: true, but when we don't people ask us about centos 6.0 :(

**Comment by SvenDowideit** [comment]
ach, yup, I'll update to make that more vague :) merci

File: docs/sources/installation/centos.md (hunk 1 above)
Comment (vbatts) [inline docs/sources/installation/centos.md]: Ah, I see. Perhaps a vague "this version or higher" ...

**Comment by SvenDowideit** [comment]
@fredlf @jamtur01 @lsm5

**Comment by jamtur01** [comment]
LGTM

**Comment by lsm5** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**